docs/guides/javascript/reactive/index.md

```diff
@@ -146,25 +146,35 @@ export default class extends BaseComponent {
 
 Any component belongs to one DOM element and is registered in a Reactive Instance. This means that to create a new instance we need to provide at least those two elements. For example:
 
-<details>
-  <summary>View example</summary>
-  <div>
+<Tabs>
+  <TabItem value="init-js-module" label="AMD module" default>
 
 ```js title="path/to/plugin/amd/src/main.js"
 import {reactiveInstance} from 'YOUR_PLUGIN/reactive';
 import {YourComponent} from 'YOUR_PLUGIN/yourcomponent;
```

Comment:
```suggestion
import YourComponent from 'YOUR_PLUGIN/yourcomponent;
```
As far as I understand, here has to be imported the component itself as the component does not export a named object, but a default class? I had to change it like that in my test project to make it work.